app/controllers/obituaries/covid_controller.rb

```diff
@@ -1,4 +1,5 @@
 class Obituaries::CovidController < ApplicationController
   def index
-  end 
+    @obituaries = Obituary.where(covid: true) 
+  end
```

Comment:
Now that this is built I'm wondering if it needs an entire controller. Not really sure how else to do it following ReSTful conventions.

Reply:
I think this is fine in its own controller. It probably wouldn't be too much in the other obituaries controller right now, but as we start to expand the search functionality I bet it would get out of hand quick.